src/pages/Detail/Detail.js

```diff
@@ -1,10 +1,255 @@
 import React, { Component } from 'react';
+import './Detail.scss';
 
 export class Detail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productInfo: [],
+      buttonValue: true,
+      buttonDropDown: true,
+      secondDropDown: true,
+      productName: '제품 이름',
+      productPrice: 0,
+      productColor: '색상 이름',
+      quantityCalculation: 1,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http:/data/data.json')
```

Comment:
에러발생할 것 같네요!! 확인해보셨나요?
```suggestion
    fetch('/data/data.json')
```
이렇게 적어주셔야지 현재 실행되고 있는 포트로 인식이됩니다! 그리고 data.json이란 파일명보다는 어떤 데이터인지 알아볼 수 있는 파일명으로 지어주시는게 좋습니다!! 모든 이름들은 다 명확하게!